### PR TITLE
cf-remote: Fixed package selection for new package names

### PR DESCRIPTION
**Pull request by olehermanse**
Tested on Ubuntu 18, Ubuntu 16, CentOS 7, CentOS 6

### CODE REVIEW

File: contrib/cf-remote/cf_remote/remote.py [hunk 1]
```diff
@@ -97,6 +97,48 @@ def get_info(host, *, users=None, connection=None):
...
+                # Examples:
+                # CentOS release 6.10 (Final)
+                # Red Hat Enterprise Linux release 8.0 (Ootpa)
+                before, after = redhat_release.split(" release ")
+                distro = "rhel"
+                if before.lower().startswith("centos"):
+                    distro = "centos"
+                major = after.split(".")[0]
+                tags.append(distro + major)
+                if distro != "el":
```
Comment (vpodzime) [inline contrib/cf-remote/cf_remote/remote.py]: `distro` cannot be equal to `"el"` here, right?

File: contrib/cf-remote/cf_remote/remote.py [hunk 2]
```diff
@@ -97,6 +97,48 @@ def get_info(host, *, users=None, connection=None):
...
+                    tags.append("el" + major)
+                if distro != "rhel":
```
Comment (vpodzime) [inline contrib/cf-remote/cf_remote/remote.py]: And here it can only be `"centos"` if it is not `"rhel"`.

File: contrib/cf-remote/cf_remote/remote.py [hunk 3]
```diff
@@ -97,6 +97,48 @@ def get_info(host, *, users=None, connection=None):
...
+                    tags.append("rhel" + major)
+
+                tags.append(distro)
+                if distro != "rhel":
+                    tags.append("rhel")
+                if distro != "el":
+                    tags.append("el")
```
Comment (vpodzime) [inline contrib/cf-remote/cf_remote/remote.py]: Same here.

File: contrib/cf-remote/cf_remote/remote.py [hunk 4]
```diff
@@ -190,6 +232,7 @@ def install_host(
         extension = None
         if "package_tags" in data and "msi" in data["package_tags"]:
             extension = ".msi"
+            del data["package_tags"]["msi"]
```
Comment (vpodzime) [inline contrib/cf-remote/cf_remote/remote.py]: What does this have to do with CentOS 6?

Comment (olehermanse) [reply contrib/cf-remote/cf_remote/remote.py]: `data["package_tags"]` is now appended (extending) tags below, to get the tags from os-release stuff. As a side effect, it means that the msi tag would be added "twice" both as an extension and as a tag. This seemed unnecessary.

File: contrib/cf-remote/cf_remote/remote.py (hunk 1 above)
Comment (olehermanse) [reply contrib/cf-remote/cf_remote/remote.py]: Right, I was just following a pattern. Fixed.

File: contrib/cf-remote/cf_remote/remote.py (hunk 2 above)
Comment (olehermanse) [reply contrib/cf-remote/cf_remote/remote.py]: Yes, I guess my thought here was more like: don't add `"rhel"` twice to the list. I tried looking for a unique list / ordered set in python, but it seems it's not part of the standard library.

Comment (olehermanse) [reply contrib/cf-remote/cf_remote/remote.py]: Fixed / tried to make code more clear.
